corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -514,8 +514,7 @@ def get_cached_payload(self):
             return CachedResponse(None)
 
         if self.sync_log:
-            stream = STREAM_RESTORE_CACHE.enabled(self.domain)
-            payload = self.sync_log.get_cached_payload(self.version, stream=stream)
+            payload = self.sync_log.get_cached_payload(self.version, stream=True)
```

Comment:
not really, its just passing the stream param through to `couchdbkit.Document.fetch_attachment`. I think its still useful to have the kwarg in the `get_cached_payload` function for tests / debugging.